### ISSUE TITLE
Tutorials

### ISSUE THREAD

**Issue by pedromorgan** [npiganeau/ts2#26]
We need a tutorial section that explain signalling..

There are a few resources out there..
- but we need to make a basic olde sginalman..
- and a local line with a single track..
- and siding and interchange

So levels...
1) newbie + up down...
2) single line controller
3) dual line controller
4) section controller
5) next.. we can go nuts..

## Sidings
So we can have a simple "coal line"
and assembling a train.. 
reverse switch     this
confirm reverse..
and ready

## single line
target and length of train
and end problem.. snow..
but can stop into siding mid way

## Dual Line
Split and mange two section in parallel.. fast line..
two entry points

## triple line
three lines at sections.. and passing 

## etrsm3. euro control.. ATC and etc..

...

**Comment by npiganeau** [comment]
It is clear that ts2 lacks documentation.

Making videos could be an easy way to have it, in the same way as http://www.railwayoperationsimulator.com/ did. At least:
- A video showing the basics of how to play
- A video showing the basics of creating a new simulation.

**Comment by ELDHOSE-P-A** [comment]
I totally agree with it. ts2 really need some basic documentation that would help the users know how to do basic stuff like setting a path, using the editor, altering the train schedules, etc...
